libs/model/src/test/seedDb.ts

```diff
@@ -58,105 +43,97 @@ export const seedDb = async (debug = false): Promise<void> => {
+    });
 
-    const nodes: Record<string, ChainNodeAttributes> = {
-      edgeware: {
-        url: 'mainnet1.edgewa.re',
-        name: 'Edgeware Mainnet',
-        balance_type: BalanceType.Substrate,
-      },
-      ethereum: {
-        url: 'https://eth-mainnet.alchemyapi.io/v2/dummy_key',
-        name: 'Ethereum Mainnet',
-        eth_chain_id: 1,
-        balance_type: BalanceType.Ethereum,
-      },
-      sepolia: {
-        id: 1263,
-        url: 'https://ethereum-sepolia.publicnode.com',
-        name: 'Sepolia Testnet',
-        eth_chain_id: 11155111,
-        balance_type: BalanceType.Ethereum,
-      },
-      osmosis: {
-        url: 'https://rpc-osmosis.ecostake.com',
-        name: 'Osmosis',
-        balance_type: BalanceType.Cosmos,
-        cosmos_chain_id: 'osmosis',
-        bech32: 'osmo',
-      },
-      csdkBeta: {
-        url: 'https://cosmos-devnet-beta.herokuapp.com/rpc',
-        name: 'Cosmos SDK v0.45.0 devnet',
-        balance_type: BalanceType.Cosmos,
-        alt_wallet_url: 'https://cosmos-devnet-beta.herokuapp.com/lcd/',
-        cosmos_chain_id: 'csdkbetaci',
-        bech32: 'cosmos',
-      },
-      csdkV1: {
-        url: 'https://cosmos-devnet.herokuapp.com/rpc',
-        name: 'Cosmos SDK v0.46.11 devnet',
-        balance_type: BalanceType.Cosmos,
-        alt_wallet_url: 'https://cosmos-devnet.herokuapp.com/lcd/',
-        cosmos_chain_id: 'csdkv1',
-        bech32: 'cosmos',
-        cosmos_gov_version: CosmosGovernanceVersion.v1,
-      },
-      csdkBetaLocal: {
-        url: 'http://localhost:5050/rpc',
-        name: 'CI: Cosmos SDK v0.45.0 devnet',
-        balance_type: BalanceType.Cosmos,
-        alt_wallet_url: 'http://localhost:5050/lcd/',
-        cosmos_chain_id: 'csdkbetalocal',
-        bech32: 'cosmos',
-        cosmos_gov_version: CosmosGovernanceVersion.v1beta1,
-      },
-      csdkV1CLocal: {
-        url: 'http://localhost:5051/rpc',
-        name: 'CI: Cosmos SDK v0.46.11 devnet',
-        balance_type: BalanceType.Cosmos,
-        alt_wallet_url: 'http://localhost:5051/lcd/',
-        cosmos_chain_id: 'csdkv1local',
-        bech32: 'cosmos',
-        cosmos_gov_version: CosmosGovernanceVersion.v1,
-      },
-      ethermintLocal: {
-        url: 'http://localhost:5052/rpc',
-        name: 'CI: Ethermint devnet',
-        balance_type: BalanceType.Cosmos,
-        alt_wallet_url: 'http://localhost:5052/lcd/',
-        cosmos_chain_id: 'evmosdevlocal',
-        bech32: 'cosmos',
-      },
-    };
-    const [
-      edgewareNode,
-      mainnetNode,
-      testnetNode,
-      osmosisNode,
-      csdkBetaNode,
-      csdkV1Node,
-      csdkBetaLocalNode,
-      csdkV1LocalNode,
-      ethermintLocalNode,
-    ] = await models.ChainNode.bulkCreate(Object.values(nodes));
+    const [edgewareNode] = await seed('ChainNode', {
+      url: 'mainnet1.edgewa.re',
+      name: 'Edgeware Mainnet',
+      balance_type: BalanceType.Substrate,
+      contracts: [],
+    });
```

Comment:
I suppose bulk insert isn't really that important for seeding, but it might make test runs a bit faster

Reply:
Are we looking for performance or convenience? The legacy seedDb util is creating a bunch of records that are not always used, and we are coupling tests to hardcoded data (ids) deeply hidden inside the utility. This PR is proposing a more explicit and direct way of testing, using your new mocked `seed` util plus deep patching abilities... we will discuss higher level seed patterns soon.